cmake/FindSndfile.cmake

```diff
@@ -13,6 +13,6 @@ find_library(SNDFILE_LIBRARY NAMES sndfile
     HINTS $ENV{SNDFILE_ROOT}/lib)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(SNDFILE DEFAULT_MSG SNDFILE_LIBRARY SNDFILE_INCLUDE_DIR)
+find_package_handle_standard_args(Sndfile DEFAULT_MSG SNDFILE_LIBRARY SNDFILE_INCLUDE_DIR)
```

Comment:
this will define a variable Sndfile_FOUND (variables are case sensitive!)

Reply:
No. It will populate both of them according to the docs: https://cmake.org/cmake/help/v3.17/module/FindPackageHandleStandardArgs.html

> FOUND_VAR <result-var>
>
>    Obsolete. Specifies either `<PackageName>_FOUND` or `<PACKAGENAME>_FOUND` as the result variable. This exists only for compatibility with older versions of CMake and is now ignored. Result variables of both names are always set for compatibility.